ui/app/components/app/connected-accounts-list/connected-accounts-list-item/connected-accounts-list-item.component.js

```diff
@@ -0,0 +1,55 @@
+import classnames from 'classnames'
+import PropTypes from 'prop-types'
+import React, { PureComponent } from 'react'
+import Identicon from '../../../ui/identicon'
+
+export default class ConnectedAccountsListItem extends PureComponent {
+  static contextTypes = {
+    t: PropTypes.func.isRequired,
+  }
+
+  static propTypes = {
+    address: PropTypes.string.isRequired,
+    className: PropTypes.string,
+    name: PropTypes.node.isRequired,
+    status: PropTypes.node.isRequired,
+    options: PropTypes.node,
+  }
+
+  static defaultProps = {
+    className: null,
+    options: null,
+  }
+
+  render () {
+    const {
+      address,
+      className,
+      name,
+      status,
+      options,
+    } = this.props
+
+    return (
+      <div className={classnames('connected-accounts-list__row', className)}>
+        <div className="connected-accounts-list__row-content">
+          <Identicon
+            className="connected-accounts-list__identicon"
+            address={address}
+            useBlockie
```

Comment:
:eyes:

Reply:
Hah, oops! Just realized I never explained this - I talked to Whymarrh about this on a call.

For posterity: `useBlockie` should be set according to the user's preferences (`metamask.useBlockie`). It was accidentally set to true here in all cases.

"Blockie" is how we used to generate identicons.